File: TypeProviders.CSharp.BuildTimeGeneration.RunTime/JsonProviderAttribute.fs
namespace TypeProviders.CSharp.BuildTimeGeneration

open System
open System.Diagnostics
open CodeGeneration.Roslyn

[<AttributeUsage(AttributeTargets.Class, Inherited = false, AllowMultiple = false)>]
[<CodeGenerationAttribute("TypeProviders.CSharp.BuildTimeGeneration.JsonProviderGenerator, TypeProviders.CSharp.BuildTimeGeneration, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null")>]
[<Conditional("CodeGeneration")>]
type JsonProviderAttribute(data: string) =
    inherit Attribute()

File: TypeProviders.CSharp.BuildTimeGeneration.Attributes/JsonProviderAttribute.cs
using CodeGeneration.Roslyn;
using System;
using System.Diagnostics;

namespace TypeProviders.CSharp.BuildTimeGeneration.Attributes
{
    [AttributeUsage(AttributeTargets.Class, Inherited = false, AllowMultiple = false)]
    [CodeGenerationAttribute("TypeProviders.CSharp.BuildTimeGeneration.JsonProviderGenerator, TypeProviders.CSharp.BuildTimeGeneration, Version=" + Globals.AssemblyVersion + ", Culture=neutral, PublicKeyToken=null")]
    [Conditional("CodeGeneration")]
    public class JsonProviderAttribute : Attribute
    {
        public JsonProviderAttribute(string data)
        {
        }
    }
}
